src/wrapper/jnienv.rs

```diff
@@ -117,6 +117,23 @@ impl<'a> JNIEnv<'a> {
         );
         Ok(class)
     }
+	
+	/// Define a new java class using an AutoByteArray
+	pub fn define_class_bytearr<S>(&self, name: S, loader: JObject<'a>, buf: AutoByteArray) -> Result<JClass<'a>>
```

Comment:
Better naming here? Not sure, maybe something like `define_class_autobytearray`?